proton-j/src/test/resources/org/apache/qpid/proton/engine/impl/ssl/cert.pem.txt

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please provide instructions on exactly how to (re)create the SSL resources used for the tests, so they can be easily updated later as needed.

E.g like the existing proton SSL test resource details: https://github.com/apache/qpid-proton/blob/master/tests/python/proton_tests/ssl_db/README.txt
Or the qpid-jms SSL test resource details (which can be sourced and execute directly): https://github.com/apache/qpid-jms/blob/master/qpid-jms-client/src/test/resources/README.txt